INTITA/JSONFiles/CountryService.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Countries.swift
+//  INTITA
+//
+//  Created by Stepan Niemykin on 27.01.2021.
+//
+
+import Foundation
+
+class CountryService {
+    func getCountries() -> [CountryModel]? {
```

Comment:
I suppose this model CountryService should have the private property: a dictionary of instances of CountryModel like this var dict : [String : String] = ["UA" : "*istanceNameOfCountryModel*"], wherefrom you can get 1 country throw func getCountries() like this: var country = dict["UA"]. To get localized in switch: dict["UA"]?.titleUA